refactron/core/pipeline.py

```
@@ -0,0 +1,197 @@
+        self.store.save(session)
+
+    def _find_fixer_name(self, issue: CodeIssue) -> Optional[str]:
+        if not self.fix_engine.can_fix(issue):
+            return None
+        for name, fixer in self.fix_engine.fixers.items():
+            try:
+                result = fixer.preview(issue, "x = 1\n")
+                if result.success:
+                    return name
+            except Exception:
+                continue
```

Ruff (0.15.9)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)